Rakefile

```diff
@@ -49,20 +50,8 @@ all_components.each do |c|
   end
 end
 
-def proxy_components
-  %w(ood-portal-generator mod_ood_proxy ood_auth_map nginx_stage).map {|name| BUILD_ROOT.join(name) }
-end
-
-proxy_components.each do |build_root|
-  file build_root => 'nginx_stage/lib/nginx_stage/version.rb' do
-    rm_rf build_root if build_root.directory?
-    cp_r build_root.basename, build_root
-  end
-end
-
-
 desc "Build OnDemand"
-task :build => all_components.map(&:build_root) + proxy_components
+task :build => all_components.map(&:build_root)
```

Comment:
@johrstrom it looks like removing the step for recursively copying the proxy components to the build directory when running `rake build` solved the problem when building in GitLab. What are the ramifications for your Ansible code and Morgan's build from source directions on https://osc.github.io/ood-documentation/master/installation/from-source.html?

Reply:
Just have to add 4 steps AFTER `rake build`.

```
rake build
cp -r mod_ood_proxy /opt/ood/
cp -r nginx_stage /opt/ood/
cp -r ood-portal-generator /opt/ood/
cp -r ood_auth_map /opt/ood/
```

Because `rake build` uses rsync with --delete it has to be done first.

Reply:
Sorry updating my comment, I meant `rake install`.

Reply:
If we really want to keep the steps simple we could make `rake install` do a long rsync command with 5 sources all going to single destination.

Reply:
This doesn't matter till we release 1.7. I say we merge if this works and open an issue to revisit the build/install etc. process, whether we want to expand rake install to have a long rsync or do something else altogether, such as drop a separate build directory and just build the apps in place.